app_feup/lib/model/entities/Course.dart

```diff
@@ -0,0 +1,48 @@
+class Course {
+  int id;
+  int fest_id;
+  String name;
+  String abbreviation;
+  String currYear;
+  int firstEnrollment;
+  String state = "";
+
```

Comment:
Is this class ever instantiated and then its values previously changed? If not, couldn't these be final?

Reply:
Made them final.